### DISCUSSION TITLE
Blurry terminal

### DISCUSSION THREAD

**Discussion by RadioactivePlays** [termux/termux-app#3515] [Q&A]
How to get that blurry terminal as shown in the README.md of the project?
![image](https://github.com/termux/termux-app/assets/19830027/4a56a4ab-cee1-443a-b243-e0679dda3a9b)

**Comment by twaik** [comment]
That screenshot is from README of termux-monet project, it is not related to official termux.

**Reply by RadioactivePlays** [reply]
Oh mb
